main.go

```diff
@@ -250,7 +246,11 @@ func main() {
 		/* Create GitCommit Info */
 		gci := new(model.GitCommitInfo)
 
-		validate(c.String("number"))
+		if !validate(c.String("number")) {
```

Comment:
MatchString returns `bool` type and `error` type.
In golang, I suppose that it is preferable to implement following.
```
matched , error := validate(c.String("number"))

if matched != false or error != nil {
  ...
}
```

Reply:
ref: https://golang.org/pkg/regexp/#MatchString

Reply:
@chaspy  Uoops, I'm sorry to misunderstand spec of `MatchString` function.

`MatchSring` function has overload.

1. `func MatchString(pattern string, s string) (matched bool, err error)`
1. `func (re *Regexp) MatchString(s string) bool`

You choiced second pattern, so could you revert?